File: GARI_Umbria_2026-08-03/DotNet/AgronicaCoreStd/AgronicaCoreModelsSTD/metaschema/CodiciNazioniISO3166.cs
using AgronicaCoreModelsSTD.baseClass;
using System;
using System.Collections.Generic;
using System.Text;

namespace AgronicaCoreModelsSTD.metaschema
{
    public class CodiciNazioniISO3166 : BaseCodeDescrStr
    {
        //public string codice { get; set; }
        //public string descrizione { get; set; }-
        public string codiceNumerico { get; set; }
        public string codiceAlpha3 { get; set; }
        public int gestioneGerarchia { get; set; }

        public CodiciNazioniISO3166() : base("", "")
        {

        }

        public CodiciNazioniISO3166(string codice) : base(codice, "")
        {

        }

        public CodiciNazioniISO3166(string codice, string descrizione) : base(codice, descrizione)
        {

        }
    }
}

File: GARI_Umbria_2026-08-03/GiasNG/src/app/Model/metaschema/CodiciNazioniISO3166.ts
export class CodiciNazioniISO3166 {
    codice: string;
    descrizione: string;
    codiceNumerico: string;
    codiceAlpha3: string;
    gestioneGerarchia: number;

    constructor(codice: string, descrizione: string, codiceNumerico: string, codiceAlpha3: string, gestioneGerarchia: number) {
        this.codice = codice;
        this.descrizione = descrizione;
        this.codiceNumerico = codiceNumerico;
        this.codiceAlpha3 = codiceAlpha3;
        this.gestioneGerarchia = gestioneGerarchia;
    }
}
